framework/include/multiapps/MultiApp.h

```diff
@@ -424,6 +433,11 @@ class MultiApp : public MooseObject, public SetupInterface, public Restartable
 
   /// Storage for command line arguments
   const std::vector<std::string> & _cli_args;
+
+  bool _keep_solution_during_restore;
```

Comment:
I was going to argue that this should be const, but looking through the source, I can see that it can be overridden by the master app. That should be noted somewhere.